frontend/src/OnLoadActions.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import GenericModal from 'components/common/GenericModal';
+import { useHistory } from 'react-router-dom';
+import { PARCEL_STORAGE_NAME, clearStorage, isStorageInUse } from 'utils/storageUtils';
+
+const OnLoadActions: React.FC = () => {
+  const history = useHistory();
+
+  return (
+    <GenericModal
+      title="Unsaved Draft"
+      message="You have an unsaved Property Draft, would you like to resume editing it?"
+      cancelButtonText="Discard"
+      okButtonText="Resume Editing"
+      display={
+        history.location.pathname !== '/submitProperty' && isStorageInUse(PARCEL_STORAGE_NAME)
```

Comment:
quick check - weren't we going to rename `/submitProperty` to `/property` ? or that's in another PR?

Reply:
I asked Jonny to do this originally.

Reply:
okay